client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -30,9 +29,9 @@ export default function Dashboard(): JSX.Element {
     getAllUserBoards();
   }, []);
 
-  useEffect(() => {
-    initSocket();
-  }, [initSocket]);
+  // useEffect(() => {
+  //   initSocket();
+  // }, [initSocket]);
```

Comment:
Can you uncomment these lines before merging?

Reply:
Yes! I didn't realize I left them commented out. The socket error was getting annoying.

Reply:
Actually, scratch that. I'm going to remove this and the imports for sockets since this is leftover from the starter code and we're not currently using it.